src/components/ThumbnailImage.js

```diff
@@ -33,73 +31,71 @@ const defaultProps = {
     imageHeight: 200,
 };
 
-class ThumbnailImage extends PureComponent {
-    constructor(props) {
-        super(props);
+/**
+ * Compute the thumbnails width and height given original image dimensions.
+ *
+ * @param {Number} width - Width of the original image.
+ * @param {Number} height - Height of the original image.
```

Comment:
NAB, Let's also describe third param - windowHeight?

Reply:
Did we disable lint for JSDoc? Those kind of issues were being caught by the linter

Reply:
Yeah, I think so. I haven't seen any error for not adding param JSDoc.
Anyways, NAB. But it think it would be good to have